adsf/lib/adsf/server.rb

```diff
@@ -2,10 +2,11 @@
 
 module Adsf
   class Server
-    def initialize(root:, live: false, index_filenames: ['index.html'], host: '127.0.0.1', port: 3000, handler: nil)
+    def initialize(root:, live: false, index_filenames: ['index.html'], auto_extensions: [], host: '127.0.0.1', port: 3000, handler: nil)
```

Comment:
This line is now excessively long. What's your preferred style for breaking long parameter lists in method decls?

Reply:
It really is growing rather large! I wouldn’t mind keeping it the way it is, but I think I prefer to split it up, like so:

```ruby
    def initialize(
      root:,
      live: false,
      index_filenames: ['index.html'],
      auto_extensions: [],
      host: '127.0.0.1',
      port: 3000,
      handler: nil,
    )
```